workers/unity/Assets/Playground/Scripts/Metrics/MetricSendSystem.cs

```diff
@@ -54,6 +55,12 @@ protected override void OnUpdate()
         private static float DefaultLoadCalculation(float fps)
         {
             float targetFps = Application.targetFrameRate;
+
+            if (targetFps == -1)
+            {
+                targetFps = DefaultTargetFrameRate;
```

Comment:
This doesn't seem like it would give you the correct load.

Reply:
Why not? You can try calculating the load with fps = 30 and 60 and then at 30 load is 1 and at 60 its 0.